amenity-points.mss

```diff
@@ -186,6 +186,13 @@
     marker-clip: false;
   }
 
+  [feature = 'amenity_public_bath'][zoom >= 17] {
+    marker-file: url('symbols/public_bath.svg');
+    marker-fill: @culture;
```

Comment:
I would rather make it `@amenity-brown`. I wouldn't put it in the same basket as cinema for example, so general category is safe. If we have sauna and shower icons (which is perfectly possible), we can make a separate category.